### PR TITLE
VIA Support: Super and Sweet 16 

### PR DESCRIPTION
**Pull request by mechmerlin**
## Description

Add VIA keymap to the Super and Sweet 16 boards. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by mechmerlin** [comment]
Failing test is with ridingwithtraffic's proton c keymap.

File: keyboards/1upkeyboards/sweet16/keymaps/via/keymap.c [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        KC_NO,    KC_NO,    KC_NO,    KC_NO,
+        KC_NO,    KC_NO,    KC_NO,    KC_NO,
+        KC_NO,    KC_NO,    KC_NO,    KC_NO,
+        KC_NO,    KC_NO,    KC_NO,    KC_NO
+    ),
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+        case UP_URL:
```
Comment (fauxpark) [inline keyboards/1upkeyboards/sweet16/keymaps/via/keymap.c]: Does this work in VIA?

Comment (Wilba6582) [reply keyboards/1upkeyboards/sweet16/keymaps/via/keymap.c]: User defined keycodes aren't in VIA yet. VIA would just show the keycode in hex.

Comment (fauxpark) [reply keyboards/1upkeyboards/sweet16/keymaps/via/keymap.c]: So, should it be removed, or?

Comment (Wilba6582) [reply keyboards/1upkeyboards/sweet16/keymaps/via/keymap.c]: I don't think it matters, it's not referenced in the keymap.

File: keyboards/1upkeyboards/sweet16/keymaps/via/keymap.c [hunk 2]
```diff
@@ -0,0 +1,60 @@
+#include QMK_KEYBOARD_H
+
+enum custom_keycodes {
+  UP_URL = SAFE_RANGE
+};
+
```
Comment (fauxpark) [inline keyboards/1upkeyboards/sweet16/keymaps/via/keymap.c]:
```suggestion
```

**Pull request closed without merging**